airlock/business_logic.py

```diff
@@ -230,6 +241,14 @@ def get_file_metadata(self, relpath: UrlPath) -> FileMetadata | None:
     def get_workspace_state(self, relpath: UrlPath) -> WorkspaceFileState | None:
         """Get workspace state of file."""
 
+    def get_request_state(self, relpath: UrlPath) -> RequestFileReviewStatus | None:
```

Comment:
nit: `get_request_status` for consistency with `RequestFileReviewStatus`?

Reply:
ah, yes, good point